GitHubUserSearch/GitHubUserSearch/Model/Model.swift

```diff
@@ -40,19 +43,19 @@ class Model {
    var currentUser:UserData?
    lazy var repositories:[RepositoryData] = [RepositoryData]()
    
-   //MARK: - memory
+   // MARK: - memory
    
    func didReceiveMemoryWarning() {
       cache.removeAllObjects()
```

Comment:
To avoid crashes you need also call super on this place.

Reply:
my bad.